Test failure in CI build 3635

The following test appears to have failed:

[#3635](https://circleci.com/gh/cockroachdb/cockroach/3635):

```
I0615 18:23:05.402465     498 range_command.go:817] initiating a split of range=7 ["db19" - "db1:") at key "db19/table"
I0615 18:23:05.403708     498 range_command.go:754] range 62: new leader lease replica 1:1 00:00:00.000 +1.000s
I0615 18:23:05.462174     498 range_command.go:754] range 63: new leader lease replica 1:1 00:00:00.000 +1.000s
I0615 18:23:05.463276     498 range_command.go:817] initiating a split of range=63 ["db19/table" - "db1:") at key "db19/tablf"
I0615 18:23:05.517083     498 range_command.go:754] range 64: new leader lease replica 1:1 00:00:00.000 +1.000s
--- FAIL: TestStoreRangeManySplits (3.78s)
	client_split_test.go:583: expected splits not found: client_split_test.go:582: condition failed to evaluate true within 1s
=== RUN TestStoreStatus
I0615 18:23:05.955945     498 rocksdb.go:88] opening in-memory rocksdb instance
I0615 18:23:05.961900     498 range.go:932] gossiping cluster id  from store 1, range 1
I0615 18:23:05.966070     498 range_command.go:754] range 1: new leader lease replica 1:1 00:00:00.000 +1.000s
I0615 18:23:05.966385     498 range.go:993] gossiping accounting config from store 1, range 1
I0615 18:23:05.967003     498 range.go:993] gossiping permissions config from store 1, range 1
I0615 18:23:05.967553     498 range.go:993] gossiping zones config from store 1, range 1
I0615 18:23:05.968288     498 range.go:940] gossiping sentinel from store 1, range 1
I0615 18:23:05.968728     498 range.go:944] gossiping first range from store 1, range 1
--
E0615 18:23:06.070524     498 queue.go:266] failure processing range range=2 ["b" - "\xff\xff") from replicate queue: allocator.go:207: unable to allocate a target store for {[] }
E0615 18:23:06.081437     498 queue.go:266] failure processing range range=1 ["" - "b") from replicate queue: allocator.go:207: unable to allocate a target store for {[] }
--- PASS: TestStoreStatus (0.13s)
=== RUN: ExampleAllocatorRebalancing
--- PASS: ExampleAllocatorRebalancing (0.97s)
FAIL
FAIL	github.com/cockroachdb/cockroach/storage	28.183s
=== RUN TestBatchBasics
I0615 18:23:32.180490     618 rocksdb.go:88] opening in-memory rocksdb instance
I0615 18:23:32.181500     618 rocksdb.go:113] closing in-memory rocksdb instance
--- PASS: TestBatchBasics (0.01s)
=== RUN TestBatchGet
I0615 18:23:32.185933     618 rocksdb.go:88] opening in-memory rocksdb instance
I0615 18:23:32.186320     618 rocksdb.go:113] closing in-memory rocksdb instance
--- PASS: TestBatchGet (0.00s)
=== RUN TestBatchMerge
I0615 18:23:32.190241     618 rocksdb.go:88] opening in-memory rocksdb instance
I0615 18:23:05.402465     498 range_command.go:817] initiating a split of range=7 ["db19" - "db1:") at key "db19/table"
I0615 18:23:05.403708     498 range_command.go:754] range 62: new leader lease replica 1:1 00:00:00.000 +1.000s
I0615 18:23:05.462174     498 range_command.go:754] range 63: new leader lease replica 1:1 00:00:00.000 +1.000s
I0615 18:23:05.463276     498 range_command.go:817] initiating a split of range=63 ["db19/table" - "db1:") at key "db19/tablf"
I0615 18:23:05.517083     498 range_command.go:754] range 64: new leader lease replica 1:1 00:00:00.000 +1.000s
--- FAIL: TestStoreRangeManySplits (3.78s)
	client_split_test.go:583: expected splits not found: client_split_test.go:582: condition failed to evaluate true within 1s
=== RUN TestStoreStatus
I0615 18:23:05.955945     498 rocksdb.go:88] opening in-memory rocksdb instance
I0615 18:23:05.961900     498 range.go:932] gossiping cluster id  from store 1, range 1
I0615 18:23:05.966070     498 range_command.go:754] range 1: new leader lease replica 1:1 00:00:00.000 +1.000s
I0615 18:23:05.966385     498 range.go:993] gossiping accounting config from store 1, range 1
I0615 18:23:05.967003     498 range.go:993] gossiping permissions config from store 1, range 1
I0615 18:23:05.967553     498 range.go:993] gossiping zones config from store 1, range 1
I0615 18:23:05.968288     498 range.go:940] gossiping sentinel from store 1, range 1
I0615 18:23:05.968728     498 range.go:944] gossiping first range from store 1, range 1
--
E0615 18:23:06.070524     498 queue.go:266] failure processing range range=2 ["b" - "\xff\xff") from replicate queue: allocator.go:207: unable to allocate a target store for {[] }
E0615 18:23:06.081437     498 queue.go:266] failure processing range range=1 ["" - "b") from replicate queue: allocator.go:207: unable to allocate a target store for {[] }
--- PASS: TestStoreStatus (0.13s)
=== RUN: ExampleAllocatorRebalancing
--- PASS: ExampleAllocatorRebalancing (0.97s)
FAIL
FAIL	github.com/cockroachdb/cockroach/storage	28.183s
=== RUN TestBatchBasics
I0615 18:23:32.180490     618 rocksdb.go:88] opening in-memory rocksdb instance
I0615 18:23:32.181500     618 rocksdb.go:113] closing in-memory rocksdb instance
--- PASS: TestBatchBasics (0.01s)
=== RUN TestBatchGet
I0615 18:23:32.185933     618 rocksdb.go:88] opening in-memory rocksdb instance
I0615 18:23:32.186320     618 rocksdb.go:113] closing in-memory rocksdb instance
--- PASS: TestBatchGet (0.00s)
=== RUN TestBatchMerge
I0615 18:23:32.190241     618 rocksdb.go:88] opening in-memory rocksdb instance

```
Please assign, take a look and update the issue accordingly.

--- a/storage/client_split_test.go
+++ b/storage/client_split_test.go
@@ -579,7 +579,7 @@ func TestStoreRangeManySplits(t *testing.T) {
 			keys = append(keys, r.Key)
 		}
 		return reflect.DeepEqual(keys, expKeys)
-	}, 1*time.Second); err != nil {
+	}, 5*time.Second); err != nil {
 		t.Errorf("expected splits not found: %s", err)
 	}
 }

